debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PgTypeName.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.postgresql;
+
+/**
+ * Contains Postgres specific datatype names, especially ones without a reliable oid.
+ *
+ * @author Olavi Mustanoja (tilastokeskus@gmail.com)
+ */
+public final class PgTypeName {
+
+    public static final String CITEXT = "citext";
```

Comment:
How about adding this to `TypeRegistry` instead? There are some type name constants there already and we handle similar cases (see `geometryOid`etc.). Let's handle all that similarly.

Reply:
Yeah that makes sense. I kind of skipped it since I couldn't wrap my head around it at first.